doonut-core/src/test/java/com/doonutmate/doonut/member/service/MemberBusinessServiceTest.java

```diff
@@ -104,6 +115,34 @@ void updateMarketingReceiveConsent() {
         assertThat(actual.marketingReceiveConsent()).isTrue();
     }
 
+    @Test
+    void deleteMember() {
```

Comment:
테스트 코드가 길어진 것 같은데, 간단하게 설명을 추가하는건 어떨까요?  
```suggestion
    @Test
    @DisplayName("멤버가 회원탈퇴하면 캘린더, 챌린지가 모두 삭제처리된다.")
    void deleteMember() {
```

Reply:
좋은생각인것 같습니다!